integrations/clickup/main.py

```diff
@@ -0,0 +1,91 @@
+from enum import StrEnum
+from typing import Any
+from clickup.client import ClickupClient
+from loguru import logger
+from port_ocean.context.ocean import ocean
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+
+class ObjectKind(StrEnum):
+    TEAM = "team"
+    PROJECT = "project"
+    ISSUE = "issue"
+
+
+async def init_client() -> ClickupClient:
+    client = ClickupClient(
+        ocean.integration_config["clickup_base_url"],
+        ocean.integration_config["clickup_personal_token"],
+    )
+    await client.initialize_team_key()  # Make sure team_key is initialized
```

Comment:
This is not a recommended way to running integrations. Please change your logic

Reply:
Logic has been changed as suggested. I have fixed the issue by using @cache_iterator_result() to keep the teams in cache